compliance/src/main/java/bio/terra/pearl/compliance/SyncVantaUsers.java

```diff
@@ -170,17 +241,22 @@ private <T extends VantaObject> Collection<T> collectAllData(WebClient wc, Param
         return elements.get();
     }
 
-    private WebClient getWebClientForIntegration(String accessToken, String integrationId, String resourceKind) {
-        WebClient.Builder webClientBuilder = WebClient.builder().defaultHeaders(h -> {
+    private WebClient.Builder getWebClientForUrl(String accessToken, String url) {
```

Comment:
Factored out a separate method so that the `patch` request can use the same kind of webClient as the `get` requests that list information about computers, git users, etc.